spec/requests/default_namespace_spec.rb

```diff
@@ -4,7 +4,19 @@
 
   include Rails.application.routes.url_helpers
 
-  let(:resource) { ActiveAdmin.register Category }
+  before do
+    load_defaults!
+    ActiveAdmin.register(Category, namespace: [:foo, :bar])
+    reload_routes!
+  end
```

Comment:
I'm not familiar with this, can you explain a little more?

Reply:
This is registering the page in nested namespace before the test run, which is required for the test below.